### PR TITLE
feat: Quiz pages

### PR DESCRIPTION
**Pull request by Shivani-1524**
Added Quiz Pages UI 

Files To Review:
QuizPages Folder
Utils/getReqests.js
Context/quiz-context.js

### CODE REVIEW

**Comment by varun-Tad** [comment]
Code looks good to me. Really like the way you styled the quiz headings too.Good to merge

File: src/Pages/QuizPages/Components/Question.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+const Question = ({ props }) => {
+    const { } = { props }
+    return (
+        <div className="quiz-container">
+            <p className="rg-title new-font bold mb-20">Question 3.</p>
+            <p className="md-title question mb-20 ml-20"> What does Viper Not say?</p>
+            <ul className="options pos-rel dark-txt mg-t-40">
+                <li className="first solid">
+                    <input type="radio" id="q3-op1" name="q3-ans" />
+                    <label for="q3-op1">
```
Comment (veereshbv04) [inline src/Pages/QuizPages/Components/Question.js]: change for to htmlFor. As for is javscript keyword we get warning

Comment (Shivani-1524) [reply src/Pages/QuizPages/Components/Question.js]: done

**Comment by Shivani-1524** [comment]
> Code looks good to me. Really like the way you styled the quiz headings too.Good to merge

Thank you varun

**Pull request closed without merging**